charts/smarter-device-manager/values.yaml

```diff
@@ -0,0 +1,72 @@
+image:
+  repository: registry.gitlab.com/arm-research/smarter/smarter-device-manager
+  tag: v1.1.2
+  pullPolicy: IfNotPresent
+
+# Override default configuration
+# config:
+#   - devicematch: ^snd$
+#     nummaxdevices: 20
+#   - devicematch: ^ttyUSB.*$
+#     nummaxdevices: 1
+
+hostNetwork: true
+dnsPolicy: ClusterFirstWithHostNet
+
+devicePluginPath: "/var/lib/kubelet/device-plugins"
+# devPath: "/special-dev/"
+# sysPath: "/special-sys/"
+
+envValueFrom:
+  NODE_NAME:
+    fieldRef:
+      apiVersion: v1
+      fieldPath: spec.nodeName
+
+hostname: smarter-device-management
+
+priorityClassName: system-node-critical
+
+# DaemonSet labels
+controllerLabels:
+  role: agent
```

Comment:
Is this required for the chart to be functional? If it is not required I would suggest commenting this out by default and putting a comment why a user would want to enable this.

Reply:
Did not find any use in the code. Will kill.